macros/PG.pl

```diff
@@ -307,7 +307,15 @@ sub ANS_NUM_TO_NAME {
 }
 
 sub store_persistent_data {
-	$PG->store_persistent_data(@_);    #needs testing
+	$PG->store_persistent_data(@_);
```

Comment:
It would be better to write this as
```suggestion
	my ($label, @values) = @_;
	$PG->store_persistent_data($label, @values);
```
perlcritic will complain about this, but furthermore it makes it clear what the expected arguments are.  This is something that needs to be changed throughout the pg code in numerous places, and has already cause much confusion on how to properly call methods.  It is simple to just pass `@_`, but forces one wanting to use the method to track down the correct arguments by tracing back through the code.